requirements.txt

```diff
@@ -11,14 +13,19 @@ Django==2.1.7
 django-cors-headers==2.5.2
 django-extensions==2.1.6
 django-heroku==0.3.1
+django-rest-swagger==2.2.0
```

Comment:
@dorothyas Please add the requirements to pipfile and run `pipenv run pipenv_to_requirements` to update the `requirements.txt` file

Reply:
@ja-odur Do I need to push the requirements-dev.txt file that is generated when I run the command above?